cogboard-webapp/src/components/widgets/dialogFields/JiraBucketsInput.js

```diff
@@ -0,0 +1,184 @@
+import React, { useState } from 'react';
+import {
+  FormControl,
+  IconButton,
+  List,
+  ListItem,
+  ListItemSecondaryAction,
+  ListItemText
+} from '@material-ui/core';
+import DeleteIcon from '@material-ui/icons/Delete';
+import EditIcon from '@material-ui/icons/Edit';
+import CheckIcon from '@material-ui/icons/Check';
+import { remove } from 'ramda';
+import { v4 } from 'uuid';
+import { prepareChangeEvent } from './helpers';
+import AddIcon from '@material-ui/icons/Add';
+import Input from '@material-ui/core/Input';
+import Fab from '@material-ui/core/Fab';
+import styled from '@emotion/styled/macro';
+
+const StyledInput = styled(Input)`
+  margin-top: 16px;
+  margin-bottom: 8px;
+`;
+const StyledFab = styled(Fab)`
```

Comment:
Use line separation between all **not oneliner** constants

Reply:
Done